frontend/src/main.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="@angular/localize" />
```

Comment:
I had to add localize again in order to use NgBootstrap's toasts. Apparently, the template NgBootstrap uses contains an `i18n-` attribute.